src/layouts/BaseLayout.astro

```diff
@@ -78,7 +78,7 @@ const navigation = Navigation.create({ baseUrl: Astro.url.origin, content: guide
         <button
           id="freshdesk"
           type="button"
-          class="btn bg-interactive-quaternary fixed right-6 bottom-6 border rounded-full shadow-lg flex justify-center w-16 h-16 hover:bg-interactive-quaternary-hover active:bg-interactive-quaternary-active"
+          class="btn space-x-0 bg-interactive-quaternary fixed right-6 bottom-6 border rounded-full shadow-lg flex justify-center w-16 h-16 hover:bg-interactive-quaternary-hover active:bg-interactive-quaternary-active"
```

Comment:
The icon in the support button was bugging me too so I've snuck this in here

Reply:
Good, that's been triggering me for ages 😅